### PR TITLE
fix #31570 anbariloche.com.ar

### LINKED ISSUE #31570: anbariloche.com.ar
### Issue URL (Anti Adblock Script)
[https://www.anbariloche.com.ar/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.anbariloche.com.ar%2F)
### Comment
Username: @alfiler
> 
> --
> La información debajo fue generada automáticamente, no hay que borrarla.
> 
> Custom userscripts:
> https://cdn.adguard.com/public/Userscripts/AdguardPopupBlocker/2.5/popupblocker.user.js
> https://cdn.adguard.com/public/Userscripts/AdguardAssistant/4.2/assistant.user.js
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Win
AdGuard version: | 6.4.1814.4903
Browser: | Chrome
AdGuard driver: | TDI
Filters: | Filter unblocking search ads and self-promotion,<br>AdGuard Base,<br>AdGuard Tracking Protection,<br>AdGuard Social Media,<br>AdGuard Spanish/Portuguese,<br>AdGuard Annoyances
Other extensions used: | adguard extension

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/31570

### CODE REVIEW

**Review by BlazDT — CHANGES_REQUESTED** [review]
Please also add this rule to hide popup ad/overlay:
.contenedor_banner_popup

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -63,6 +63,7 @@ moveout.in##.post_share
 celtabettv3.com,turkbet.tv,pasizle5.com,hiltonbettv1.com##.channel-share
 cosmopolitan.de##.sharingList
 turksemedia.nl##.share_box_container
+anbariloche.com.ar##.widget_twitter
```
Comment (BlazDT) [inline SocialFilter/sections/specific.txt]: Please also block this:
||platform.twitter.com/js/timeline$domain=anbariloche.com.ar

Either with $domain suffic or third-party suffix.

File: SocialFilter/sections/social_trackers.txt [hunk 2]
```diff
@@ -83,6 +83,7 @@
 ||pinterest.com/v1/urls/count.json$third-party
 ||platform.linkedin.com^$third-party
 ||platform.stumbleupon.com/*/widgets.js$third-party
+||platform.twitter.com/js/timeline$third-party
```
Comment (Alex-302) [inline SocialFilter/sections/social_trackers.txt]: it is not a tracker. Please take a look into the script.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/popups.txt [hunk 3]
```diff
@@ -90,6 +90,7 @@ fforfree.net###popupFbLikeBox
 oneplus.com#$#.dialog-lock { overflow:visible!important; position:relative!important; top:0px!important; }
 oneplus.com#$##header-auto-subscribe { display:none!important; }
 15min.lt###feedback-btn
+anbariloche.com.ar##.contenedor_banner_popup
```
Comment (BlazDT) [inline AnnoyancesFilter/sections/popups.txt]: It is an ad popup, please move.

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/popups.txt (hunk 3 above)
Comment (zloyden) [reply AnnoyancesFilter/sections/popups.txt]: Please, tell me where exactly it should be?

Comment (BlazDT) [reply AnnoyancesFilter/sections/popups.txt]: Spanish/specific

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**